docs/pages/foundations/messaging/components.js

```diff
@@ -0,0 +1,222 @@
+    text="Saved"
+    thumbnail={
+      <Image
+        alt="Modern ceramic vase pin."
+        naturalHeight={564}
+        naturalWidth={564}
+        src="https://i.ibb.co/Lx54BCT/stock1.jpg"
+      />
+    }
+    thumbnailShape="circle"
+  />
+  <Toast
+    text="You’re not connected to the internet"
+    variant="error"
+  />
+</Flex>
+`}
+        />
+        <Flex direction="column" gap={4}>
+          <Text>
+            Toasts are the least disruptive of messages in that they are ephemeral and don’t require
+            a user to act or dismiss them. They appear opposite a surface’s main navigation and
+            overlay content without fully blocking it. They can be used to:
+          </Text>
+          <Text>
+            <ul>
+              <li>Acknowledge a user action immediately after it happens</li>
+              <li>Acknowledge a user action while nudging them to improve their experience</li>
+              <li>
+                Alert users of connectivity issues or unknown errors without disrupting their flow
+              </li>
+            </ul>
+          </Text>
+          <Text>
+            <Link href="https://gestalt.pinterest.systems/web/toast" underline="always">
+              Go to the Toast component
+            </Link>
+          </Text>
+        </Flex>
+      </MainSection>
+      <MainSection name="What component should I use?">
+        <Flex direction="column" gap={4}>
+          <Text>
+            Here is a quick diagram to help you make a decision on what Messaging component to use.
+            If you don’t see it here, perhaps you need a related component.
+          </Text>
+
+          <Box>
+            <iFrame
+              style={{ border: 'none' }}
+              width="100%"
+              height="450"
+              src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FXxaqjsGExBDWWs7HDRhaYW%2FMessaging-decision-tree-H2%3Fnode-id%3D102%253A1092"
+              allowFullScreen
+            />
+          </Box>
+
+          <Text>
+            <Link
+              href="https://www.figma.com/file/XxaqjsGExBDWWs7HDRhaYW/Messaging-decision-tree-H2?node-id=102%3A1092"
+              underline="always"
+              externalLinkIcon="default"
+            >
+              View in Figma
+            </Link>
+          </Text>
+        </Flex>
+      </MainSection>
+      <MainSection name="Related">
+        <MainSection.Subsection
+          description={`
+**[Popover](/web/popover)**
+A popover can be formatted to provide help and guidance around new features, or features a user may not be aware of.
+
+**[Upsell](/web/upsell)**
+Upsells are on-page banners that encourage more ad spend and upgrades. They can also be used to promote and market new features.
+
+**Error message variants**
+Toast provides temporary feedback on an interaction. Toasts appear at the bottom of a desktop screen or top of a mobile screen, instead of blocking the entire page.
```

Comment:
This messaging about Toasts seems out of place?

Reply:
Thanks! Done!